src/Symfony/Bridge/Monolog/Handler/ChromePhpHandler.php

```diff
@@ -31,6 +31,8 @@ class ChromePhpHandler extends BaseChromePhpHandler
 
     /**
      * Adds the headers to the response once it's created.
+     *
+     * @param FilterResponseEvent $event
```

Comment:
@HugoSantiagoBecerraAdan Same feeling as https://github.com/symfony/symfony/pull/29518#issuecomment-445468954. Symfony does not add phpdoc annotations which does not bring some value, and this one provides nothing more than the typehint itself.